mono/mini/mini-arm64.h

```diff
@@ -19,6 +19,13 @@
 
 #define MONO_MAX_IREGS 32
 #define MONO_MAX_FREGS 32
+#define MONO_MAX_XREGS 0
+#define MONO_ARCH_CALLEE_SAVED_XREGS 0
+#define MONO_ARCH_CALLEE_XREGS 0
```

Comment:
```
#define MONO_ARCH_CALLEE_XREGS MONO_ARCH_CALLEE_FREGS
#define MONO_ARCH_CALLEE_SAVED_XREGS MONO_ARCH_CALLEE_SAVED_FREGS
```

may have to be reordered in the file to place where FREGs are defined. The registers are mirrored, ie. XREGs and FREGs are the same ones on the hardware.

Aside from that, the ABI states: "Additionally, only the bottom 64 bits of each value stored in v8-v15 need to be preserved [7]; it is the responsibility of the caller to preserve larger values.". I am not sure if we will need to adjust for that? Maybe `MONO_ARCH_CALLEE_SAVED_XREGS` should be 0 after all?

Reply:
Ref: https://developer.arm.com/docs/ihi0055/d/procedure-call-standard-for-the-arm-64-bit-architecture#id47